fem/fespace.cpp

```diff
@@ -357,10 +357,11 @@ void FiniteElementSpace::BuildDofToArrays()
       const int n = elem_dof -> RowSize(i);
       for (int j = 0; j < n; j++)
       {
-         if (dof_elem_array[dofs[j]] < 0)
+         int dof = (dofs[j] >= 0) ? dofs[j] : - dofs[j] - 1;
```

Comment:
```suggestion
         int dof = DecodeDof(dofs[j]);
```

A bit more self-documenting this way